.github/workflows/ci.yml

```diff
@@ -14,59 +14,77 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: olafurpg/setup-scala@v13
+
+      - uses: actions/setup-java@v3
         with:
-          java-version: graalvm-ce-java11@21.1.0
+          distribution: 'temurin' 
+          java-version: '11'
+
       - name: Check formatting
         run: ./bin/scalafmt --test
+
   integrations:
     name: Build integrations tests
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macOS-latest]
-        jdk: [graalvm-ce-java11@21.1.0]
+        jdk: [11, 17]
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: true
-      - uses: olafurpg/setup-scala@v13
+
+      - uses: actions/setup-java@v3
         with:
+          distribution: 'temurin' 
           java-version: ${{ matrix.jdk }}
+
       - name: Tests
         run: |
-          ./bin/sbt-ci.sh \
+          sbt \
               "mavenBloop/publishM2" \
               "mavenBloop/test" \
               "gradleBloop211/compile" \
               "gradleBloop212/compile" \
               "gradleBloop212/test" \
               "sbtBloop10Shaded/publishLocal" \
               "sbtBloop10/scripted"
-        shell: bash
+
   bridges:
     name: Test platform bridges
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        jdk: [graalvm-ce-java11@21.1.0]
+        jdk: [11, 17]
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: true
-      - uses: olafurpg/setup-scala@v13
+
+      - name: Cache SBT
```

Comment:
https://github.com/coursier/cache-action

Reply:
For some reason (may be it's just me) but this cache action occasionally has **very** long restore times.
It's sporadic, but I've seen it on many builds

Reply:
Ah really? I've never had an issue with it.

Reply:
If it's just me I'll revert this 👍

Reply:
Just ignore me idn. I'm just a sucker for the one liner to add it in and forget about it. But who knows, maybe this whole time I've been having slow restore times and I never noticed it.